pom.xml

```diff
@@ -54,30 +54,32 @@
   </scm>
 
   <properties>
+    <globalVersion>5-SNAPSHOT</globalVersion>
+
```

Comment:
@jdcasey Need to version the dependency on the plugin but can't use `project.version` within inheritance. This means we need to make sure the release plugin updates this value. Could we do `mvn -DdryRun=true release:prepare -DglobalVersion=5` ?

Reply:
👍 if that works.